src/gluonts/model/gpvar/_network.py

```diff
@@ -348,3 +348,19 @@ def hybrid_forward(
             past_is_pad=past_is_pad,
             future_time_feat=future_time_feat,  # (batch_size, prediction_length, num_features)
         )
+
+    def post_process_samples(self, samples: Tensor) -> Tensor:
+        """
+        Identity map.
+
+        Parameters
+        ----------
+        samples
+            Tensor of shape (num_parallel_samples*batch_size, 1, target_dim)
+
+        Returns
+        -------
+            Tensor of samples with the same shape.
+
+        """
+        return samples
```

Comment:
Isn’t this already defined as the identity in the superclass?

Reply:
It was not earlier; moved the method to `DeepVARNetwork` from `DeepVarPredictionNetwork`. Now this is not longer required. Thanks!